apps/web/scripts/kernel/handlers/common.ts

```diff
@@ -3,9 +3,10 @@ import { modules } from '@walless/ioc';
 import { ResponseCode } from '@walless/messaging';
 import type { PublicKeyDocument } from '@walless/store';
 import { selectors } from '@walless/store';
+import { extensionActions } from 'state/extension';
```

Comment:
`kernel` should not contain any module from the extension itself (e.g this case is `state/extension`), it make bundled `background-script.js` file bigger than necessary.